privacyidea/lib/config.py

```diff
@@ -124,11 +124,13 @@ def reload_from_db(self):
                         "Description": sysconf.Description}
                 for resolver in Resolver.query.all():
                     resolverdef = {"type": resolver.rtype,
-                                   "resolvername": resolver.name}
+                                   "resolvername": resolver.name,
+                                   "__CENSOR_KEYS__": []}
```

Comment:
I chose this on purpose. Imho it is internal and I wanted to avoid any conflict with potential new resolvers.

Reply:
Hm, but how could a conflict arise? Doesn't the ``resolverdef`` dictionary only contain the keys ``type``, ``resolvername`` and ``data`` (and ``__CENSOR_KEYS__``) in any case?

I think this is not entirely internal, because the standard ``GET /resolver`` endpoint now doesn't return passwords anymore, and the ``__CENSOR_KEYS__`` information will be useful for users of our REST API in order to know which keys are censored.

Reply:
interesting aspect. Will change the key accordingly.

Reply:
done in 76ae8895

Reply:
:+1: